build-tools/packages/build-cli/src/instructionalPromptWriter.ts

```diff
@@ -51,8 +51,14 @@ interface Section {
  * Map release groups to ADO pipeline
  */
 export const ADOPipelineLinks = new Map<ReleasePackage | ReleaseGroup | undefined, string>([
-	[MonoRepoKind.Client, "https://dev.azure.com/fluidframework/internal/_build?definitionId=12"],
-	[MonoRepoKind.Server, "https://dev.azure.com/fluidframework/internal/_build?definitionId=30"],
+	[
```

Comment:
I think it's because they prioritize line length over consistency for the most part. If I rename `MonoRepoKind.Azure` to
`MonoRepoKind.Azure1`, it's broken into two lines.